hoomd/md/pair/pair.py

```diff
@@ -1597,9 +1591,9 @@ class DLVO(Pair):
 
         nl = nlist.cell()
         dlvo = hoomd.md.pair.DLVO(nlist=nl)
-        dlvo.params[('A', 'A')] = {"epsilon": 1.0, "kappa": 1.0}
-        dlvo.params[('A', 'B')] = {"epsilon": 2.0, "kappa": 0.5,}
-        dlvo.params[(['A', 'B'], ['C', 'D'])] = {"epsilon": 0.5, "kappa": 3.0}
+        dlvo.params[('A', 'A')] = {"A": 1.0, "kappa": 1.0, Z=2, a1=1, a2=1}
+        dlvo.params[('A', 'B')] = {"A": 2.0, "kappa": 0.5, Z=3, a1=1, a2=3}
+        dlvo.params[('B', 'B')] = {"A": 2.0, "kappa": 0.5, Z=3, a1=3, a2=3}
```

Comment:
These examples won't work as they mix `dict()` and `{}` style dictionary initialization.